garak/probes/_tier.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tier: model importance level of probes
+
+
+Tier 1: Of concern	
+
+Low pass rate or low z-score may be problematic	
+
+(1) Examine probe results; 
+(2) Flag to security/alignment team; 
+(3) consider adding to model card.
+
+
+Tier 2: Compete with state of the art
+
+Low z-score may be problematic
+
+(1) Examine probe results. 
+(2) Consider adding to model card.
+
+
+Tier 3: Informational
+
+Context sensitive; low scores may affect suitable use cases	
+
+If you know a probe fits your a particular use case, treat as Tier 2
+
+
+Tier 9	Unlisted
+
+Duplicated, deprecated, fluctuating, or non-adversarial probe. 	
+
+OK to ignore for now, unless there's a content safety issue not covered by the content safety assessments
+"""
+
+from enum import IntEnum
+
+
+class Tier(IntEnum):
+    OF_CONCERN = 1
+    COMPETE = 2
```

Comment:
These names are hard to understand at a glance. While I think `OF_CONCERN` imparts the message it needs to, `COMPETE` is very unclear on first glance.

Something like `COMPETE_WITH_SOTA` or `COMPARE_WITH_SOTA`, this might infer that all probes in this tier are in the z-score set (bag) though.

Thinking on how this might be _used_ as a spec criteria in the future may also help find a good name, consider if I wanted to pass `-p tier:compete` what do you think this would select, all `OF_CONCERN` + `COMPETE`, just `COMPETE`?

Might be thinking too far ahead thought this seem like a logical usage for `Tier`.